config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -281,6 +281,44 @@ presubmits:
         env:
         - name: GOPATH
           value: /go
+  - name: pull-kubernetes-node-kubelet-serial-containerd
+    always_run: false
+    optional: true
+    skip_report: false
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-kubelet
+      testgrid-tab-name: pr-node-kubelet-serial-containerd
```

Comment:
Side note: at some point I wanna split all the presubmits into a separate tab

Reply:
Agree for later.